xml/common_criteria.xml

```diff
@@ -188,7 +188,7 @@
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Service Isolation&mdash;Run Different Services on Separate
+     <term>Service isolation&mdash;run different services on separate
      Systems</term>
```

Comment:
Here it spans to another line and the Systems is still capitalized.

Reply:
Please fix this manually, then. This is something I cannot easily update within the script.